packages/fxa-auth-server/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "strictNullChecks": true,
```

Comment:
Isn't this is already covered by `_dev/tsconfig.base.json` with `"strict"`?

Reply:
Yes - also was not intentional that this was left in.